### PR TITLE
Mark "Talk" active on /call/token URLs

### PR DESCRIPTION
**Pull request by nickvergessen**
On backport to stable17 the class name needs to be changed, so manual backporting it is.

### Steps
1. Open Talk
2. Go to a conversation
3. Reload the page
4. Try to find the "active triangle" on the top navigation bar

### CODE REVIEW

**Review by rullzer — APPROVED** [review]
sure

**Review by Ivansss — APPROVED** [review]
👍

**Comment by rullzer** [comment]
Ha... PHAN is not happy here: https://drone.nextcloud.com/nextcloud/server/23079/5/6

**Review by ChristophWurst — APPROVED** [review]

**Comment by rullzer** [comment]
Phan still not happy...

File: lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php [hunk 1]
```diff
@@ -128,6 +128,11 @@ public function beforeController($controller, $methodName) {
 		// for normal HTML requests and not for AJAX requests
 		$this->navigationManager->setActiveEntry($this->appName);
 
+		// @suppress PhanUndeclared
```
Comment (georgehrke) [inline lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php]:
```suggestion
		// @suppress PhanUndeclaredClassConstant
```

Comment (nickvergessen) [reply lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php]: Fixed

Comment (ChristophWurst) [reply lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php]: Still showing a
```
lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php:132 PhanUndeclaredClassConstant Reference to constant class from undeclared class \OCA\Talk\Controller\PageController 
```

File: lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php [hunk 2]
```diff
@@ -128,6 +128,11 @@ public function beforeController($controller, $methodName) {
 		// for normal HTML requests and not for AJAX requests
 		$this->navigationManager->setActiveEntry($this->appName);
 
+		/** @suppress PhanUndeclaredClassConstant */
```
Comment (nickvergessen) [inline lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php]: > lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php:132 PhanUndeclaredClassConstant Reference to constant class from undeclared class \OCA\Talk\Controller\PageController 

I give up understanding this....

Comment (kesselb) [reply lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php]: > Phan issue types may be suppressed by adding @suppress <issue_type_list> to the doc block of the closest function, method or class scope to where the issue is emitted.

https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#suppress

**Comment by nickvergessen** [comment]
Oh thanks @kesselb

**Pull request merged**